newIDE/app/src/AssetStore/index.js

```diff
@@ -344,6 +355,18 @@ export const AssetStore = React.forwardRef<Props, AssetStoreInterface>(
       [shouldAutofocusSearchbar]
     );
 
+    React.useLayoutEffect(
```

Comment:
just read about this effect. (and the fact it's not recommended, as it can block rendering)
When using classic `useEffect()` I suspect you get a small layout shift? Is it that bad ?

Reply:
> When using classic useEffect() I suspect you get a small layout shift? Is it that bad ?

Indeed, I get a layout shift because the filters close just after the rendering of the homepage. It does not look super great so I think it was enough to use this (and yes it would be better to not use this and have the whole state updated at once, but it seems hard to do properly :/)